sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/NicIpConfiguration.java

```diff
@@ -322,7 +322,7 @@ interface WithSubnet<ParentT> {
          *     definition
          */
         interface WithPublicIPAddress<ParentT>
-            extends HasPublicIpAddress.UpdateDefinitionStages.WithPublicIPAddress<WithAttach<ParentT>> {
+            extends NetworkInterface.WithDefinitionSecondaryOrUpdatePublicIPAddress.UpdateDefinitionStages.WithPublicIPAddress<WithAttach<ParentT>> {
```

Comment:
This is a breaking change to existing interface.

Reply:
In `NetworkInterface.WithDefinitionSecondaryOrUpdatePublicIPAddress`, only the interface that needs to pass `DeleteOptions` is described, so it is necessary to extends `HasPublicIPAddress` at the same time.